app/views/halls/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: @hall do |form| %>
+  <% if @hall.errors.any? %>
```

Comment:
An if statement like this can be quite useful in certain situations (for example when you want to display information that there are no errors), however in this context it seems to be redundant. 

`@hall.errors.each` will simply do nothing if there are no errors, so there is no need to check if there are any errors.

Reply:
Didn't think of that, thanks!